src/Filters/atoms/DateTimeBlock.tsx

```diff
@@ -142,13 +176,25 @@ const DateTimeBlock : React.FC<IProps> = ({
   useEffect(()=>{
     if(allowAfterMidnight && isEqual(date, endOfDay(date))){
       setDisplayHours('24');
-      setDisplayMinutes('00');
-    } else {
-      setDisplayMinutes(format(date, 'mm'));
-      setDisplayHours(format(date,'HH'));
+      setDisplayMinutes('0');
```

Comment:
I think this should be `00` .

Reply:
For initial I am showing `0` in every input. so I have use `0`. and I also use blur event. so when blur event runs it will show `00` in input.

Reply:
ok